jobs/ci-kubernetes-e2e-gce-sd-logging.env

```diff
@@ -0,0 +1,9 @@
+### job-env
+PROJECT=k8s-jkns-gce-sd-log
```

Comment:
You mean [check_projects.py](https://github.com/kubernetes/test-infra/blob/master/jenkins/check_projects.py)? If so, yes, I did